src/test/java/com/shima/todo_list/mapper/TodoListMapperTest.java

```diff
@@ -59,6 +61,42 @@ class TodoListMapperTest {
         Optional<TodoList> todoLists = todoListMapper.findById(5);
         assertThat(todoLists).isEmpty();
     }
+
+
+    @Test
+    @Sql(
+            scripts = {"classpath:/sqlannotation/delete-todolists.sql", "classpath:/sqlannotation/insert-todolists.sql"},
+            executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD
+    )
+
+    //CREATE機能のDBテスト
+    @Transactional
+    void 新規のタスクを登録すること() {
+        TodoList todoList = new TodoList("リリーステスト", LocalDate.of(2023, 12, 10), null, null);
+        todoListMapper.insert(todoList);
+        List<TodoList> todoLists = todoListMapper.findAll();
+        assertThat(todoLists).hasSize(5)
+                .contains(
+                        new TodoList(1, "構想", LocalDate.of(2023, 12, 6), null, null),
+                        new TodoList(2, "API作成", LocalDate.of(2023, 12, 7), null, null),
+                        new TodoList(3, "テスト", LocalDate.of(2023, 12, 8), null, null),
+                        new TodoList(4, "リリース", LocalDate.of(2023, 12, 9), null, null),
+                        todoList
+                );
+    }
+
+    @Test
+    @DataSet(value = "datasets/todolists.yml")
+    @ExpectedDataSet(value = "datasets/insert_todolists.yml", ignoreCols = "id")
+
+    //CREATE機能のDBテスト(DBRider)
+    @Transactional
+    void 新規のタスクを登録することver1() {
```

Comment:
元々のテストを削除して、テストケースはそのままにしましょう。
```suggestion
    void 新規のタスクを登録すること() {
```

Reply:
こちらの対応でよろしいでしょうか。

94d43d587bbce5a12d4f9395a04f1a42238946c5

### 動作確認


![A3F70803-6DD4-4CB7-A070-5386449AE2E5](https://github.com/simamumu89/Todo_API/assets/142807674/019909d5-7044-41d6-87ba-ff70cc7067ae)